README.md

```diff
@@ -0,0 +1,49 @@
+[![Build Status](https://travis-ci.org/mayflower/libholiday.svg?branch=master)](https://travis-ci.org/mayflower/libholiday)
+
+A library to calculate holidays.
+
+Supported countries and states
+
+* America
+* Germany
+  * Hamburg, Saarland, BadenWuerttemberg, Hesse, SaxonyAnhalt, Bavaria, Saxony, Berlin, LowerSaxony, SchleswigHolstein, Brandenburg, MecklenburgWesternPomerania, Bremen, Thuringia, NorthRhineWestphalia, RhinelandPalatinate
```

Comment:
I'm a bit unsure if those names should really be in English...

Reply:
Why not? It gives an international flare and show ambitions :=

Reply:
on the other hand, wouldn't it be sufficient to say `Germany and all federal states`?